cognite_toolkit/_cdf_tk/client/api_client.py

```diff
@@ -1,15 +1,13 @@
-from __future__ import annotations
+from typing import TYPE_CHECKING
 
-from cognite.client import ClientConfig, CogniteClient
+from cognite.client import ClientConfig
+from cognite.client._api_client import APIClient
 
-from .api.dml import DMLAPI
-from .api.location_filters import LocationFiltersAPI
-from .api.robotics import RoboticsAPI
+if TYPE_CHECKING:
+    from ._toolkit_client import ToolkitClient
```

Comment:
What's the purpose of this construct? Under what condition can this be false?

Reply:
Avoid circular imports. It is only used by static type checkers such as mypy